osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/infra/messaging/DeviceRequestMessageProcessor.java

```diff
@@ -167,30 +168,15 @@ protected void handleError(final Exception e, final MessageMetadata messageMetad
         this.responseMessageSender.send(protocolResponseMessage);
     }
 
-    protected void handleExpectedError(final Exception e, final String correlationUid,
-            final String organisationIdentification, final String deviceIdentification, final String domain,
-            final String domainVersion, final String messageType, final int messagePriority) {
-        LOGGER.error("Expected error while processing message", e);
-        final FunctionalException ex = new FunctionalException(FunctionalExceptionType.VALIDATION_ERROR,
-                ComponentType.PROTOCOL_OSLP, e);
-
-        final DeviceMessageMetadata deviceMessageMetadata = new DeviceMessageMetadata(deviceIdentification,
-                organisationIdentification, correlationUid, messageType, messagePriority);
-        final ProtocolResponseMessage protocolResponseMessage = ProtocolResponseMessage.newBuilder().domain(domain)
-                .domainVersion(domainVersion).deviceMessageMetadata(deviceMessageMetadata)
-                .result(ResponseMessageResultType.NOT_OK).osgpException(ex).build();
-
-        this.responseMessageSender.send(protocolResponseMessage);
-    }
-
     public void handleUnableToConnectDeviceResponse(final DeviceResponse deviceResponse, final Throwable t,
             final Serializable messageData, final DeviceResponseMessageSender responseMessageSender,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "responseMessageSender". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)